src/fragments/lib/auth/js/getting-started.mdx

```diff
@@ -250,4 +250,4 @@ To implement authentication flows using Amplify you can either use the Amplify U
 
 `Auth` has over 30 methods including [`signUp`](/lib/auth/emailpassword#sign-up), [`signIn`](/lib/auth/emailpassword#sign-in), [`forgotPassword`](/lib/auth/manageusers#forgot-password), and [`signOut`](/lib/auth/emailpassword#sign-out) that allow you full control over all aspects of the user authentication flow.
 
-Check out the complete API [here](https://aws-amplify.github.io/amplify-js/api/classes/authclass.html).
+Check out the complete API and docs [here](https://ui.docs.amplify.aws/components/authenticator).
```

Comment:
This change isn't needed, as "Option 2" is where customers don't use the UI component, but wire up the Auth class themselves.